tests/test_flows/test_flow_functions.py

```diff
@@ -263,6 +263,11 @@ def test_sklearn_to_flow_list_of_lists(self):
         self.assertEqual(server_flow.parameters['categories'], '[[0, 1], [0, 1]]')
         self.assertEqual(server_flow.model.categories, flow.model.categories)
 
+    def test_get_flow1(self):
+        # Make sure that issue #305 doesn't pop up any more
```

Comment:
While linking to Github issues is very useful for full context/explanation, I think it's good to accompany it where reasonable with a very brief explanation in the file. That way one has a slight idea of what's going on without leaving the editor, e.g.:

> `# Fetching a flow with no external version is allowed (see #305)`.

You can commit with `[skip ci]` to avoid a rerun for this change.